### PR TITLE
Niko/ia/faqs improvements

### PR DESCRIPTION
**Pull request by nikomancy**
# FAQ Improvements
Work on articles and information architecture and article quality.

## What's Changing and Why
1. **Removing FAQ section:** In general, FAQ sections work like junk-drawers for docs. There's usually better places to put these things.
2. **Moving Risk and Latency pages to Architecture:** These documents discuss architectural topics and end up being more like SLAs than anything else. Having read the other Architecture docs, the risk and latency docs should fit in.
3. **Turning Common Issues into Best Practices:** Each common issue is being turned into its own article on the best practices. This 
4. **Rewriting common issues:** The common issues have been edited to lead with clear statements of what they are about and why, use fewer words, have more bulleted lists, and generally be easier to read and follow.

### CODE REVIEW

**Comment by nikomancy** [comment]
I just rebased. This should solve the failing tests.

File: docs/sdks/best-practices/avoid-round-trips.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   - Pressure to implement "fetch all flags at initialization"
+   - Potential conflicts with other best practices
+   - Reduced quality of analytic data
+
+### Best Practice
+Use Eppo's client-side SDKs for most use cases. They:
+- Pre-fetch obfuscated flag configurations
+- Perform assignment logic locally
+- Evaluate flags only when features are rendered
+- Typically complete round trips in [under 20ms](/sdks/architecture/latency)
```
Comment (lukasgoetzweiss) [inline docs/sdks/best-practices/avoid-round-trips.md:31]: Maybe "typically initialize in under 20 ms"? Just to further make it clear that this round trip only needs to happen once

**Review by lukasgoetzweiss — COMMENTED** [review]
Looks good to me. Just want to spot check the deploy preview before merging

**Comment by nikomancy** [comment]
Rebasing again. Hopefully this will clear up the test issues.

**Pull request merged**